addons/ar/arca/charges.go

```diff
@@ -0,0 +1,62 @@
+package arca
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/tax"
+	"github.com/invopop/validation"
+)
+
+// Charge keys for Argentina ARCA v4
+const (
+	ChargeKeyNationalTaxes               cbc.Key = "national-taxes"
+	ChargeKeyProvincialTaxes             cbc.Key = "provincial-taxes"
+	ChargeKeyMunicipalTaxes              cbc.Key = "municipal-taxes"
+	ChargeKeyInternalTaxes               cbc.Key = "internal-taxes"
+	ChargeKeyGrossIncomeTax              cbc.Key = "gross-income-tax"
+	ChargeKeyVATPrepayment               cbc.Key = "vat-prepayment"
+	ChargeKeyGrossIncomeTaxPrepayment    cbc.Key = "gross-income-tax-prepayment"
+	ChargeKeyMunicipalTaxesPrepayment    cbc.Key = "municipal-taxes-prepayment"
+	ChargeKeyOtherPrepayments            cbc.Key = "other-prepayments"
+	ChargeKeyVATNotCategorizedPrepayment cbc.Key = "vat-not-categorized-prepayment"
+	ChargeKeyOther                       cbc.Key = "other"
```

Comment:
These feel very awkward... couldn't figure out why until I saw the extensions. My recommendation would be to avoid trying to create custom GOBL mappings and instead assume the customer will know the appropriate extension to apply. I'd stick to generic keys like `tax` and perhaps `withholding` (I'm guessing that's what is meant by `Prepayment`?) and define a subset of extensions that are determined or required for each key.

I find it easiest to imagine what this would look like in the UI and the select box that would need to be presented. You can imagine that when the charge key is `tax`, that you get a select box for the correct extension to apply.